Source/LanguageModelManager.mm

```diff
@@ -305,4 +305,28 @@ + (void)setPhraseReplacementEnabled:(BOOL)phraseReplacementEnabled
     gLanguageModelMcBopomofo.setPhraseReplacementEnabled(phraseReplacementEnabled);
 }
 
++ (nullable NSString *)pronounciationFor:(NSString *)phrase {
```

Comment:
Same here, pro_nun_ciation, but maybe also consider calling this `readingFor:` for the reason suggested in another comment.